binary_search_trees/array_to_bst.py

```diff
@@ -10,4 +10,62 @@ def arr_to_bst(arr):
         Balanced Binary Search Tree using the elements in the array.
         Return the root of the Binary Search Tree.
     """
-    pass
\ No newline at end of file
+
+    class BinarySearch_tree:
+        def __init__(self) :
+            self.root   = None  
+
+        def insert(self, value):
```

Comment:
You never call your `insert` function anywhere which is part of why many of the tests are failing. You have the logic written out, but it never gets used!